### ISSUE TITLE
[QUEST] "Mission Probable": Garrison's Command Table not working.

### ISSUE THREAD

**Issue by DivineOwner777** [AshamaneProject/AshamaneCore#757]
The quest("Horde" Mission Probable) can't be completed because the Command Table in the Frostwall/Lunarfall WOD Garrison doesn't open the mission window.

Horde and Alliance Command Tables are both broken and do not work. 

Horde Command Table: http://www.wowhead.com/npc=80432/command-table
Alliance Command Table: http://www.wowhead.com/npc=81546/command-table

**Comment by Traesh** [comment]
It's not just a npc issue, all mission system require scripting ATM

**Comment by DivineOwner777** [comment]
Hi Traesh, is there an estimate for when the missions will be scripted? :)

**Comment by martin26** [comment]
Garrison is still WIP, I dont think its a priority though since who would use Garrisons when Legion content is more important, even basic stuff like upgrading town hall to tier 2 would be ok, no need for more stuff since once you hit level 98 you better start legion content anyway.

**Comment by DivineOwner777** [comment]
Valid point Martin26, I just noticed that upgrading the garrison is impossible because the Finalize Plots function doesn't work, you basically get your Garrison, and then go out questing in Shadowmoon/Frostfire Ridge without upgrading/spawning any buildings.

**Comment by martin26** [comment]
Exactly @DivineOwner777. I guess in the future some fixes here and there will be made, but basic stuff, but I can be wrong of course, more working content is always welcome! I saw a commit about flight npc in draenor being added which is nice since it was not working.

**Comment by DivineOwner777** [comment]
That would be awesome to have the flight masters working :), Eventually having a fully working Garrison would be great(even just basic functions like you mentioned @martin26 )

I have been setting a wide range of quests to auto-complete just to get through frostfire ridge/shadowmoon lol.

Do you have a link to the Commit? :)

**Comment by martin26** [comment]
It was a trainer temporary fix and not a flight master one, but it allows you to fly around draenor which is cool, you got the Draenor Pathfinder achievement, its better than nothing.

**Comment by DivineOwner777** [comment]
Oh nice! Yeah that will help people out.

I made it so all the Flight Paths are open, haha, You could also try putting the Flight Master right outside of the garrison(I don't know if that would work though.)

Also, do you know if there is a way to talk directly to the Developers? Like a group chat or something like that?

**Comment by martin26** [comment]
Of course, here: https://discordapp.com/channels/370885128404074496/386485314912518157
